ui/apps/platform/src/Containers/NetworkGraph/deployment/DeploymentBaselinesSimulated.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import {
+    Alert,
+    AlertVariant,
+    Bullseye,
+    Divider,
+    Spinner,
+    Stack,
+    StackItem,
+    Toolbar,
+    ToolbarContent,
+    ToolbarItem,
+} from '@patternfly/react-core';
+
+import { getNumFlows } from '../utils/flowUtils';
+
+import FlowsTable from '../common/FlowsTable';
+import FlowsTableHeaderText from '../common/FlowsTableHeaderText';
+import { Flow } from '../types/flow.type';
+
+type DeploymentBaselinesSimulatedProps = {
+    deploymentId: string;
+};
+
+function DeploymentBaselinesSimulated({ deploymentId }: DeploymentBaselinesSimulatedProps) {
+    // component state
+    const networkSimulatedBaselines: Flow[] = [];
+    const isLoading = false;
+    const error = '';
+
+    const initialExpandedRows = networkSimulatedBaselines
+        .filter((row) => row.children && !!row.children.length)
+        .map((row) => row.id); // Default to all expanded
+    const [expandedRows, setExpandedRows] = React.useState<string[]>(initialExpandedRows);
+    const [selectedRows, setSelectedRows] = React.useState<string[]>([]);
```

Comment:
nit: I think our usual pattern is to also import the named `{ useState }` from the React package.

Can change in a later PR

```suggestion
    const [expandedRows, setExpandedRows] = useState<string[]>(initialExpandedRows);
    const [selectedRows, setSelectedRows] = useState<string[]>([]);
```